docs/content/rules/sort-object-types.mdx

```diff
@@ -209,16 +209,6 @@ type User = {
 
 ```
 
-### [DEPRECATED] ignorePattern
```

Comment:
Should we also update `docs/content/guide/getting-started.mdx:143-152`?
Currently, this page has an example with `ignorePattern`.